CodeOfConduct.md

```diff
@@ -4,7 +4,7 @@
 
 All members and visitors of Hobart Makers Inc. agree to this code of conduct. It helps Hobart Makers Inc. maintain a safe and inclusive environment for all.
 
-Information on how to report incidents that may be in breach of this code of conduct ... ** TODO **
+Please report incidents that may be in breach of this code of conduct to the management committee in person, by email committee@hobartmakers.com, or any other convenient method.
```

Comment:
I think a separate address can also help for making sure it gets dealt with in a more speedy fashion.

it is desirable to not be needed but if someone needs it it needs to be handled correctly and swiftly which is where not necessarily being the exact same people that make up the committee could be useful. It's likely to be a subset of people that are on the committee.